plugins/woocommerce/src/Admin/PageController.php

```diff
@@ -254,6 +254,11 @@ public function get_current_page() {
 	 * @return string Current screen ID.
 	 */
 	public function get_current_screen_id() {
+		// Return early if this is a REST API request.
+		if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
```

Comment:
nit: Looks like the function [`wp_is_serving_rest_request`](https://developer.wordpress.org/reference/functions/wp_is_serving_rest_request/) is now available for use. It was added in version 6.5.